README.md

```diff
@@ -564,29 +564,29 @@ For remote overrides, the following extended settings must be configured:
   
-  **NOTE:** MongoDB Atlas quota is for **all** databases in cluster, while each instance will get only size of **its own database only**. It is ok when you only have **one** database in cluster (most common scenario) but will not work for multiple parallel databases. In such case, spliting known quota equally beetween databases and setting `DBSIZE_MAX` to that fraction may help, but wont be precise.
+  **NOTE:** MongoDB Atlas quota is for **all** databases in cluster, while each instance will get only size of **its own database only**. It is ok when you only have **one** database in cluster (most common scenario) but will not work for multiple parallel databases. In such case, splitting known quota equally between databases and setting `DBSIZE_MAX` to that fraction may help, but won't be precise.
 
   All sizes are expressed as integers, in _Mebibytes_ `1 MiB == 1024 KiB == 1024*1024 B`
 
-  * `DBSIZE_MAX` (`496`) - Maximal allowed size of database on your mongoDB server, in MiB. You need to adjust that value to match your database hosting limits - default value is for standard Heroku mongoDB free tier.
-  * `DBSIZE_WARN_PERCENTAGE` (`60`) - Threshold to show first warning about database size. When database reach this percentage of `DBSIZE_MAX` size - pill will show size in yellow. 
-  * `DBSIZE_URGENT_PERCENTAGE` (`75`) - Threshold to show urgent warning about database size. When database reach this percentage of `DBSIZE_MAX` size, it is urgent to do backup and clean up old data. At this percentage info pill turns red.
+  * `DBSIZE_MAX` (`496`) - Maximal allowed size of database on your mongoDB server, in MiB. You need to adjust that value to match your database hosting limits - default value is for the standard Heroku mongoDB free tier.
+  * `DBSIZE_WARN_PERCENTAGE` (`60`) - Threshold to show first warning about database size. When database reach this percentage of `DBSIZE_MAX` size - will show size in yellow. 
+  * `DBSIZE_URGENT_PERCENTAGE` (`75`) - Threshold to show an urgent warning about database size. When database reaches this percentage of `DBSIZE_MAX` size, it is urgent to do backup and clean up old data. At this percentage, info pill turns red.
   * `DBSIZE_ENABLE_ALERTS` (`false`) - Set to `true` to enable notifications about database size.
   * `DBSIZE_IN_MIB` (`false`) - Set to `true` to display size of database in MiB-s instead of default percentage.
   
-  This plugin should be enabled by default, if needed can be diasabled by adding `dbsize` to the list of disabled plugins, for example: `DISABLE="dbsize"`.
+  This plugin should be enabled by default if needed can be disabled by adding `dbsize` to the list of disabled plugins, for example: `DISABLE="dbsize"`.
```

Comment:
Probably more correct to change this to "... by default. If needed, it can be disabled ..."